lametro/views.py

```diff
@@ -283,6 +284,20 @@ def get_context_data(self, **kwargs):
 
         context["USING_ECOMMENT"] = settings.USING_ECOMMENT
 
+        # Only provide notices for this event's public comment setting
+        if event.accepts_live_comment:
+            context["event_notices"] = EventNotice.objects.filter(
+                comment_conditions__contains=["accepts_live_comment"]
+            )
```

Comment:
Deciding which notices to pass through based off of public comment status in the view to take some burden off of the template.